issue 3198

schlenkibus opened the pull request:
addded 'back..' button to all applicable parameters in soundscreen submenus

closes #3198

MatthiasSeeber reviewed (changes requested):
- [x] Voices, Mono and Unison Sub Screens are good

### Scale Sub Screen:

- [ ] SB3 "back ..." leads back to (root) Sound Screen instead of leading one step back to Master Screen
- [ ] SB1 "Master ..." would then be obsolete
- [x] Base Key Parameter lacks SB3 "back ..."

The requested changes seem the most generic and simple/user-friendly to me. However, this will require higher-level approval than just mine.

### Part Screen:
- [x] Part Volume (Split/Layer), Fade From, Fade Range (both Layer) lack SB3 "back ..."

schlenkibus commented:
Crash Report:
- [ ] init sound
- [ ] hwui: single -> convert to split -> to layer -> convert to Single

schlenkibus commented:
Open Question is still how we want the Scale Layout to look like. Either allow a back.. button or a Master... one but not both.

schlenkibus commented:
Removed back.. Button from Postion C in Scale Layouts and renamed Master.. Button_A to back..
This changes were made after a discussion with Stephan.
